scenarios/aws/fakeintake/fakeintakeparams/params.go

```diff
@@ -13,17 +13,17 @@ type Option = func(*Params) error
 // NewParams returns a new instance of Fakeintake Params
 func NewParams(options ...Option) (*Params, error) {
 	params := &Params{
-		LoadBalancerEnabled: true,
+		LoadBalancerEnabled: false,
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
This will make LB for fakeintake disabled by default for all scenarios, while we would like to skip fakeintake only for VM scenarios. 

We could keep `test-infra` as is, and always apply `WithoutLoadBalancer` on stack definitions that wrap VM environments

Reply:
Since we have the current limit of 32 loadbalancer, I thought safer to disable it by default all together to assure to only use it when when needed. What do you think ? Otherwise I'll just disable it from the VM wrapper

Reply:
Already being tested in [PR#21535](https://github.com/DataDog/datadog-agent/pull/21535)

Reply:
Thanks ! 👀 I'll change that